website/docs/terms/dml.md

```diff
@@ -0,0 +1,103 @@
+
+All `SELECT` statements need three elements: a `SELECT` clause in the beginning, the actual field selection and manipulation, and a `FROM` statement which is specifying which database object you’re trying to access.
+
+Here’s an example `SELECT` statement:
+
+```sql
+select
+
+	payment_method,
+	sum(amount) AS amount
+
+from {{ ref(‘raw_payments’) }}
+group by 1
+```
+
+In this example, your selection of the `payment_method` column and summation of the `amount` column is the meat of your query. The `from {{ ref(‘raw_payments’) }}` specifies the actual table you want to do the selecting from.
+
+### INSERT
+
+Using the `INSERT` DML command, you can add rows to a table that exists in your database. To be honest, data folks are rarely inserting data into tables manually with the `INSERT` command. Instead, data team members will most often use data that’s already been inserted by an <Term id="elt" /> tool or other data ingestion process.
+
+You can insert a record for myself [in jaffle_shop’s](https://github.com/dbt-labs/jaffle_shop) `raw_customers` table like this:
+
+```sql
+INSERT INTO raw_customers VALUES (101, ‘Kira’, ‘F.’);
+```
+
+As you can see from this example, you clearly set all the column values that exist in your `raw_customers` table. For `INSERT` statements, you can explicitly specify the values you want to insert or use a query result to set the column values.
+
+### DELETE
+
+The `DELETE` command will remove rows in an existing table in your database. In practice, you will usually specify a `WHERE` clause with your `DELETE` statement to only remove specific rows from a table. But, you shouldn't really ever delete rows from tables. Instead, you should apply filters on queries themselves to remove rows from your modeling or analysis.
+
+For the most part, if you wanted to remove all existing rows in a table, but keep the underlying table structure, you would use the `TRUNCATE` DDL command. If you wanted to remove all rows and drop the entire table, you could use the `DROP` DDL command.
+
+You can delete the record for any Henry W. in jaffle_shop’s customers table by executing this statement:
+
+```sql
+DELETE FROM customers WHERE first_name = ‘Henry’ AND last_name = ‘W.’;
+```
+
+### UPDATE
+
+With the `UPDATE` statement, you can change the actual data in existing rows in a table. Unlike the `ALTER` DDL command that changes the underlying structure or naming of database objects, the `UPDATE` statement will alter the actual row-level data. You can qualify an `UPDATE` command with a `WHERE` statement to change the values of columns of only specific rows.
+
+You can manually update the status column of an order in your orders table like this:
+
+```sql
+UPDATE orders SET status = ‘returned’ WHERE order_id = 7;
```

Comment:
returned should be string (green single quote)

Reply:
Fixed